submissions/Semka094/hooli-style-popup/index.html

```diff
@@ -0,0 +1,137 @@
+                            <a href="#" class="columnItem">
+                                <div><img src="./images/photos.png" alt="Photo"></div>
+                                <div class="itemTitle">Photo</div>
+                            </a>
+                            <a href="#" class="columnItem">
+                                <div><img src="./images/apps.png" alt="Apps"></div>
+                                <div class="itemTitle">Apps</div>
+                            </a>
+                            <a href="#" class="columnItem">
+                                <div><img src="./images/documents.png" alt="Documents"></div>
+                                <div class="itemTitle">Documents</div>
+                            </a>
+                            <a href="#" class="columnItem">
+                                <div><img src="./images/chat.png" alt="Chat"></div>
+                                <div class="itemTitle">Chat</div>
+                            </a>
+                            <a href="#" class="columnItem">
+                                <div><img src="./images/classroom.png" alt="Classroom"></div>
+                                <div class="itemTitle">Classroom</div>
+                            </a>
+                            <a href="#" class="columnItem">
+                                <div><img src="./images/collection.png" alt="Collection"></div>
+                                <div class="itemTitle">Collection</div>
+                            </a>
+                            <a href="#" class="columnItem">
+                                <div><img src="./images/mars.png" alt="Mars"></div>
+                                <div class="itemTitle">Mars</div>
+                            </a>
+                            <a href="#" class="columnItem">
+                                <div><img src="./images/blogs.png" alt="Blogs"></div>
+                                <div class="itemTitle">Blogs</div>
+                            </a>
+                            <a href="#" class="columnItem">
+                                <div><img src="./images/promo.png" alt="Promos"></div>
+                                <div class="itemTitle">Promos</div>
+                            </a>
+                        </div>
+                        <div class="evenMoreWrapper">
+                            <a class="evenMoreLink" href="https://github.com/kottans">Even more from Kottans</a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="navItem icon">
+            <a class="navLink" href="#">
+                <img src="./images/bell.png" alt="bell">
+            </a>
+        </div>
+        <div class="navItem profile">
+            <a class="navLink" href="#">
+                <img src="./images/profileImg_1.png" alt="Profile image">
+            </a>
+        </div>
+    </nav>
+</div>
+
+
```

Comment:
Avoid unused lines in your code. Use them only if needed.

Reply:
Fixed.